include/EVT/io/pin.hpp

```diff
@@ -115,15 +115,26 @@ enum class Pin {
     SPI_MISO = PB_14,//D12
     SPI_MOSI = PB_15,//D11
     SPI_CS = PB_6,   //D10
+    I2C_SDA = PB_8,
+    I2C_SCL = PB_9,
 #endif
 #ifdef STM32F334x8
     SPI_SCK = PA_5, //D13
     SPI_MISO = PA_6,//D12
     SPI_MOSI = PA_7,//D11
     SPI_CS = PB_6,  //D10
+    I2C_SDA = PB_8,
+    I2C_SCL = PB_9,
 #endif
+
+#ifdef STM32F4xx
+    I2C_SDA = PB_9,
```

Comment:
Are these pins correct? They're the opposite of the pins for the F3 chips

Reply:
They are right